.isort.cfg

```diff
@@ -1,6 +1,7 @@
 [settings]
 line_length=120
-known_third_party=boto3,botocore,awscli,tabulate,argparse,configparser,pytest,pytest,pytest-datadir,pytest-html,pytest-rerunfailures,pytest-xdist,argparse,retrying,junitparser,Jinja2
+known_future_library=future
```

Comment:
Probably don't need this anymore because we are using python3 everywhere?

Reply:
Changed